### ISSUE TITLE
Need to run e2e tests with CNI enabled

### ISSUE THREAD

**Issue by douglas-reid** [istio/istio#13457]
There have been several recent issues filed that have claimed that aspects of Istio, including telemetry, aren't working when the reporter has installed Istio with CNI. In all of the scenarios, we've fallen back to relying on manual testing against a known-good cluster. IMHO, it would be much better if we had automated tests verifying behavior.

/cc @howardjohn

**Comment by howardjohn** [comment]
See https://github.com/istio/istio/pull/12529/files
and
https://github.com/istio/tools/pull/101

**Comment by douglas-reid** [comment]
@howardjohn Yup, those are steps along the path. This issue can be the umbrella tracking for the effort.
